src/main/java/org/dependencytrack/notification/vo/BomValidationFailed.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * This file is part of Dependency-Track.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) OWASP Foundation. All Rights Reserved.
+ */
+package org.dependencytrack.notification.vo;
+
+import org.dependencytrack.model.Bom;
+import org.dependencytrack.model.Project;
+import org.dependencytrack.resources.v1.problems.InvalidBomProblemDetails;
+
+public class BomValidationFailed {
+
+    private Project project;
+    private String bom;
+    private InvalidBomProblemDetails problemDetails;
```

Comment:
Consider just passing the `errors` here instead of the entire `InvalidBomProblemDetails` object. Problem details are meant specifically for API responses (hence containing a `status` field etc.), let's not mix this with other aspects of the application.